pkg/filters/filters.go

```diff
@@ -107,3 +127,47 @@ func HasIncludedAge(creationTime metav1.Time, filterOpts *Options) (bool, error)
 
 	return true, nil
 }
+
+// SystemNamespaceFilter is a filter that filters out namespaces that are created with the cluster by default
+func SystemNamespaceFilter(object runtime.Object, filterOpts *Options) bool {
+	if meta, ok := object.(metav1.Object); ok {
+		namespaceName := meta.GetName()
+		for _, systemNamespace := range SystemNamespaceNames {
+			if namespaceName == systemNamespace {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+func ExcludeNamespacesFilter(object runtime.Object, filterOpts *Options) bool {
```

Comment:
What's the purpose of the include and exclude namespace functions?

Reply:
added comments to both functions